packages/block-editor/src/store/test/selectors.js

```diff
@@ -3034,3 +3035,23 @@ describe( 'selectors', () => {
 		} );
 	} );
 } );
+
+describe( '__experimentalGetParsedReusableBlock', () => {
+	const state = {
+		settings: {
+			__experimentalReusableBlocks: [
+				{
+					id: 1,
+					content: { raw: '' },
+				},
+			],
+		},
+	};
+
+	// Regression test for https://github.com/WordPress/gutenberg/issues/26485. See https://github.com/WordPress/gutenberg/issues/26548.
+	it( "should not throw an exception if reusable block's content.raw is an empty string", () => {
+		expect( () =>
+			__experimentalGetParsedReusableBlock( state, 1 )
+		).not.toThrowError();
```

Comment:
You can also update this check and test for the actual result. If code would throw it will fail the test as well.

Reply:
Do you mean to re-purpose the test to test one of the actual scenarios/happy-path for the `__experimentalGetParsedReusableBlock` helper func? That means it wouldn't be a specific regression test anymore. That's not a big deal, but then we'd be testing the func, and not the bug regression (which would still be tested indirectly, as you mention, though). I'm fine with that but just want to make sure we're in the same page.

Reply:
Yes, this is what I meant. `not.toThrowError();` part would be replaced with the assertion checking the actual result and the description for the test updated.